libraries/chain/include/eos/chain/chain_controller.hpp

```diff
@@ -121,7 +121,7 @@ namespace eosio { namespace chain {
          block_id_type               get_block_id_for_num( uint32_t block_num )const;
          optional<signed_block>      fetch_block_by_id( const block_id_type& id )const;
          optional<signed_block>      fetch_block_by_number( uint32_t num )const;
-         const signed_transaction&    get_recent_transaction( const transaction_id_type& trx_id )const;
+         //const signed_transaction&    get_recent_transaction( const transaction_id_type& trx_id )const;
```

Comment:
Add why commented out, Maybe just // see definition for why this is commented out.